youtube_dl/extractor/tvp.py

```diff
@@ -1,139 +1,247 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals
+# -*- encoding: utf-8 -*-
+from .common import InfoExtractor, ExtractorError
 
-import re
+VIDEO_LISTING_URL = ('http://www.api.v3.tvp.pl/shared/listing.php'
+                     '?dump=json&direct=true&count=-1&parent_id={id}')
+META_URL = 'http://www.tvp.pl/shared/video_data.php?dump=json&video_id={id}'
+TOKENIZER_URL = 'http://www.tvp.pl/shared/cdn/tokenizer_v2.php?object_id={id}'
+FILE_INFO_URL = 'http://www.tvp.pl/pub/stat/videofileinfo?video_id={id}'
+IGNORED_MIMETYPES = 'application/vnd.ms-ss', 'application/x-mpegurl'
 
-from .common import InfoExtractor
+
+class TvpApi:
```

Comment:
I don't see much point extracting it in separate class. Base class inherited from `InfoExtractor` is preferred.